spec/code_hunk_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'spec_helper'
+
+RSpec.describe Airbrake::CodeHunk do
+  describe "#to_h" do
+    context "when a file is empty" do
+      subject { described_class.new(fixture_path('empty_file.rb'), 1).to_h }
+
+      it { is_expected.to eql({}) }
+    end
```

Comment:
For empty file I expect to see `{1: ""}`

Reply:
👍